openedx/core/djangoapps/user_authn/views/tests/test_login.py

```diff
@@ -1041,6 +1041,7 @@ class LoginSessionViewTest(ApiTestCase, OpenEdxEventsTestMixin):
     USERNAME = "bob"
     EMAIL = "bob@example.com"
     PASSWORD = "password"
+    REGISTER_INTENT = True
```

Comment:
its not some bool. please check it from request in tests as well as you are doing actually